app/src/main/java/com/babylon/wallet/android/presentation/transaction/composables/FeePayerSelectionContent.kt

```diff
@@ -40,26 +67,113 @@ fun LazyListScope.feePayerSelectionContent(
         Spacer(modifier = Modifier.height(RadixTheme.dimensions.paddingDefault))
     }
     items(candidates) { candidate ->
-        AccountSelectionCard(
+        Card(
```

Comment:
Detekt here is right, so what if we make the whole `Card` a separate private composable `FeePayerCandidateCard`? 
Easier to read and no need to Suppress

Reply:
Will do